### PR TITLE
bug: markdown errors

### PR DESCRIPTION
**Pull request by lucasgonze**
Fixes https://github.com/OpenChain-Project/Reference-Material/issues/35

Signed-off-by: Lucas Gonze <lucas@gonze.com>

### CODE REVIEW

File: Self-Certification/Questionnaire/ISO5230-2020/en/OpenChain Self-Certification Questionnaire 2021-11-26.md [hunk 1]
```diff
@@ -12,20 +12,19 @@ OpenChain ISO/IEC 5230:2020 is the International Standard for open source licens
...
+As part of our online support you can also self-certify using our web app for free here:<br>
+https://certification.openchainproject.org
 
-As part of our online support you can also self-certify using our web app for free here:\
-[[https://certification.openchainproject.org/]{.underline}](https://certification.openchainproject.org/)
-
-We have a video discussing online self-certification here:\
-[[https://www.youtube.com/watch?v=lVM4RH8RRl0]{.underline}](https://www.youtube.com/watch?v=lVM4RH8RRl0)
+We have a video discussing online self-certification here:<br>
+https://www.youtube.com/watch?v=lVM4RH8RRl0
```
Comment (jacobdjwilson) [inline Self-Certification/Questionnaire/ISO5230-2020/en/OpenChain Self-Certification Questionnaire 2021-11-26.md:22]: Recommend enclose in angle brackets to follow markdown guidelines.
https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses

File: Self-Certification/Questionnaire/ISO5230-2020/en/OpenChain Self-Certification Questionnaire 2021-11-26.md [hunk 2]
```diff
@@ -12,20 +12,19 @@ OpenChain ISO/IEC 5230:2020 is the International Standard for open source licens
 
 You can adopt OpenChain ISO/IEC 5230:2020 by self-certification in your own time or working with a service provider for independent assessment or third-party certification. Our recommended path is self-certification and we provide this questionnaire to support this with a series of "yes" or "no" questions.
 
-We have a lot of resources to support you if you need assistance. You can join our mailing lists, our webinars, our group calls and our regional work groups to discuss challenges with your peers and in your native language. You can get started here:
+We have a lot of resources to support you if you need assistance. You can join our mailing lists, our webinars, our group calls and our regional work groups to discuss challenges with your peers and in your native language. You can get started here:<br>
+https://www.openchainproject.org/community
 
-[[https://www.openchainproject.org/community]{.underline}](https://www.openchainproject.org/community)
+As part of our online support you can also self-certify using our web app for free here:<br>
+https://certification.openchainproject.org
```
Comment (jacobdjwilson) [inline Self-Certification/Questionnaire/ISO5230-2020/en/OpenChain Self-Certification Questionnaire 2021-11-26.md:19]: Recommend enclose in angle brackets to follow markdown guidelines.
https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses

File: Self-Certification/Questionnaire/ISO5230-2020/en/OpenChain Self-Certification Questionnaire 2021-11-26.md [hunk 3]
```diff
@@ -12,20 +12,19 @@ OpenChain ISO/IEC 5230:2020 is the International Standard for open source licens
 
 You can adopt OpenChain ISO/IEC 5230:2020 by self-certification in your own time or working with a service provider for independent assessment or third-party certification. Our recommended path is self-certification and we provide this questionnaire to support this with a series of "yes" or "no" questions.
 
-We have a lot of resources to support you if you need assistance. You can join our mailing lists, our webinars, our group calls and our regional work groups to discuss challenges with your peers and in your native language. You can get started here:
+We have a lot of resources to support you if you need assistance. You can join our mailing lists, our webinars, our group calls and our regional work groups to discuss challenges with your peers and in your native language. You can get started here:<br>
+https://www.openchainproject.org/community
```
Comment (jacobdjwilson) [inline Self-Certification/Questionnaire/ISO5230-2020/en/OpenChain Self-Certification Questionnaire 2021-11-26.md:16]: Recommend enclose in angle brackets to follow markdown guidelines.
https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses

**Review by jacobdjwilson — CHANGES_REQUESTED** [review]
Great catch, thanks for your contribution! A couple additional suggestions to align with markdown guidelines.

File: Self-Certification/Questionnaire/ISO5230-2020/en/OpenChain Self-Certification Questionnaire 2021-11-26.md (hunk 3 above)
Comment (lucasgonze) [reply Self-Certification/Questionnaire/ISO5230-2020/en/OpenChain Self-Certification Questionnaire 2021-11-26.md]: @jacobdjwilson Done

**Review by jacobdjwilson — APPROVED** [review]
Looks great!

**Comment by shanecoughlan** [comment]
@goneall I think we may need your review here :)

**Comment by goneall** [comment]
> I think we may need your review here :)

@shanecoughlan - Not really my area of expertise (Markdown), but from a quick look I'm not sure the PR will generate an email when clicking on the link since the link doesn't have a `mailto:` prefix - but someone more experience with Markdown would be much better than myself at a review.

Also - just a note - if we want to make any changes to the text that shows up in the app, the [Conformance-Questionnaire](https://github.com/OpenChain-Project/conformance-questionnaire) will need to be updated.

**Review by shanecoughlan — APPROVED** [review]
All looks great.

**Comment by shanecoughlan** [comment]
Thanks all. Looking solid and merging.

**Pull request merged**